### ISSUE TITLE
Fixing php inside javascript causes erroneous  > ! % pcs - comment - end!# -->

### ISSUE THREAD

**Issue by esetnik** [junstyle/vscode-php-cs-fixer#41]
Example:
```
        <script>
            // ### Begin: Update User Status Function
            $("input#user_active").change(function() {
                $.get("/profile/logic/SetUserProfile.php", {
                    <?php
                    if ($_COOKIE["isTO"]) {
                        ?>
                        Action: "UpdateUserDepartment",
                        UserID: "<?php echo $UserID; ?>",
                        DepartmentID: "<?php echo $userDepartmentID; ?>",
                        <?php
                    } else {
                        ?>
                        Action: "UpdateProfile",
                        UserHexID: "<?php echo $UserHexID; ?>",
                        <?php
                    }
                    ?>
                        Status: $(this).is(":checked") ? 1 : 0
                    },
                    function(data) {
                        // console.log(data);
                        // * Reload Table
                        getUsers();
                    });
            });
            // ### End: Update User Status Function
        </script>
```

Output:
```
<script>
    // ### Begin: Update User Status Function
    $("input#user_active").change(function() {
        $.get("/profile/logic/SetUserProfile.php", {
                <?php
                if ($_COOKIE["isTO"]) { ?
                    > ! % pcs - comment - end!# -->
                        Action: "UpdateUserDepartment",
                        UserID: "<?php echo $UserID; ?>",
                        DepartmentID:
                        "<?php echo $userDepartmentID; ?>",
                        <?php
                } else { ?
                    > ! % pcs - comment - end!# -->
                        Action: "UpdateProfile",
                        UserHexID:
                        "<?php echo $UserHexID; ?>",
                        <?php
                } ?
                > ! % pcs - comment - end!# -->
                Status : $(this).is(":checked") ? 1 : 0
            },
            function(data) {
                // console.log(data);
                // * Reload Table
                getUsers();
            });
    });
    // ### End: Update User Status Function
</script>
```

**Comment by junstyle** [comment]
bug fix, updated.

**Comment by esetnik** [comment]
Confirmed fixed in v0.1.62. Thank you so much!
